hera_pspec/pspecdata.py

```diff
@@ -670,22 +684,30 @@ def C_model(self, key, model='empirical', time_index=None, known_cov=None):
                 # add model to key
                 Ckey = ((dset, dset), (bl,bl), ) + (model, time_index, False, True,)
 
+        # Update self._C with known_cov
+        if known_cov is not None:
+            spw = slice(self.spw_range[0]-self.filter_extension[0], self.spw_range[1]+self.filter_extension[1])
```

Comment:
why is filter extension needed here? I think this is a dayenu specific attribute, and should probably not be used unless dayenu filtering is specifically requested. Or if by default filter_extension is set to [0, 0]. please clarify for me, thanks

Reply:
I keep the filter extension here to make sure the shape of the input covariance array for the spectral window is the same as ones obtained in other methods, such as when model is chosen as `empirical' or 'dset', we have the processed data self.x(key, include_extension=True), self.w(key, include_extension=True), which are already clipped by filter extension. And yes the default filter_extension is set to [0, 0].

Reply:
Okay, let's use the same structure as `PSpecData.x()`. Can we include a `include_extensions=False` kwarg in the function call and make this
```
if include_extensions:
    spw = slice(...
else:
    spw = slice(...
```

Reply:
Sure.